pkg/controller/core.oam.dev/v1alpha2/application/application_controller.go

```diff
@@ -101,7 +101,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		if !kerrors.IsNotFound(err) {
 			logCtx.Error(err, "get application")
 		}
-		return ctrl.Result{}, client.IgnoreNotFound(err)
+		return ctrl.Result{RequeueAfter: common2.ApplicationReSyncPeriod}, client.IgnoreNotFound(err)
```

Comment:
let's add a commno value for return.

For example:  `SuccessRequeueResult`

Reply:
Agree.